lib/ui/platform_dispatcher.dart

```diff
@@ -1768,6 +1768,852 @@ enum DisplayFeatureState {
   postureHalfOpened,
 }
 
+// This map is generated by //flutter/tools/gen_locale.dart
+// Mappings generated for language subtag registry as of 2019-02-27.
+// static const Map<String, String> _deprecatedRegionSubtagMap = <String, String>{
+//   'BU': 'MM', // Burma; deprecated 1989-12-05
+//   'DD': 'DE', // German Democratic Republic; deprecated 1990-10-30
+//   'FX': 'FR', // Metropolitan France; deprecated 1997-07-14
+//   'TP': 'TL', // East Timor; deprecated 2002-05-20
+//   'YD': 'YE', // Democratic Yemen; deprecated 1990-08-14
+//   'ZR': 'CD', // Zaire; deprecated 1997-07-14
+// };
+
+// This jump table function is 2x faster than a Map and is generated by:
+// https://gist.github.com/gaaclarke/533d5948ec242defb408eec1ce04c55f
+String _deprecatedRegionSubtagMap(String x) {
+  final List<int> units = x.codeUnits;
+  if (units.length != 2) {
+    return x;
+  }
+  switch (units[0]) {
+    case 66 /*B*/ :
+      {
+        switch (units[1]) {
+          case 85 /*U*/ :
+            return 'MM';
+        }
+        break;
+      }
+    case 68 /*D*/ :
+      {
+        switch (units[1]) {
+          case 68 /*D*/ :
+            return 'DE';
+        }
+        break;
+      }
+    case 70 /*F*/ :
+      {
+        switch (units[1]) {
+          case 88 /*X*/ :
+            return 'FR';
+        }
+        break;
+      }
+    case 84 /*T*/ :
+      {
+        switch (units[1]) {
+          case 80 /*P*/ :
+            return 'TL';
+        }
+        break;
+      }
+    case 89 /*Y*/ :
+      {
+        switch (units[1]) {
+          case 68 /*D*/ :
+            return 'YE';
+        }
+        break;
+      }
+    case 90 /*Z*/ :
+      {
+        switch (units[1]) {
+          case 82 /*R*/ :
+            return 'CD';
+        }
+        break;
+      }
+  }
+  return x;
+}
+
+// This map is generated by //flutter/tools/gen_locale.dart
+// Mappings generated for language subtag registry as of 2019-02-27.
+// static const Map<String, String> _deprecatedLanguageSubtagMap = <String, String>{
```

Comment:
nit: add something about this map being a more readable version of the jump table below.